inc/views/template_parts.php

```diff
@@ -210,6 +210,19 @@ protected function post_class( $post_id = null, $additional = '' ) {
 				$class .= ' nv-non-grid-article';
 			}
 		}
+		
+		// Filter the Core classes if the structure is not standard.
+		$default_order         = array(
+			'thumbnail',
+			'title-meta',
+			'excerpt',
+		);
+		$order                 = json_decode( get_theme_mod( 'neve_post_content_ordering', wp_json_encode( $default_order ) ) );
+		$is_thumbnail_inactive = ! in_array( 'thumbnail', $order, true );
+
+		if ( $is_thumbnail_inactive ) {
+			$class = str_replace( 'has-post-thumbnail', '', $class );
+		}
```

Comment:
@preda-bogdan, do you think moving the `$order` creation outside the function should be a better approach? Same for function `get_article_inner_content`? The value of `$order` is the same for all of them.

Reply:
@Soare-Robert-Daniel if you see opportunities for small refactors you can do them. I don't think it is necessary to ask for approval on this as the goal for all of us is to improve the code overall while fixing reported issues. 💪🏼